core/src/Kokkos_Concepts.hpp

```diff
@@ -120,6 +120,15 @@ struct LaunchBounds {
   static constexpr unsigned int minBperSM{minB};
 };
 
+#ifdef KOKKOS_ENABLE_SYCL
+template <int size>
+struct SubGroupSize {
```

Comment:
Do we want to make it `Experimental::` ?

Reply:
I'm fine either way especillay if that mean that we can evaluate its usefulness some more before committing to keeping it.